### PR TITLE
Fix missing docs and simplify period_method check

### PR DESCRIPTION
**Pull request by zilchms**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -101,11 +101,6 @@
   String $capkgs_version                   = 'present',
   String $pkg_version                      = 'present',
   Integer $agingtolerance                  = 24,
-  Boolean $nosymlinks                      = true,
-  Boolean $inet6glue                       = false,
-  Boolean $nowarnings                      = true,
-  Boolean $noerrors                        = false,
-  Boolean $randomcron                      = true,
   Optional[Stdlib::Httpurl] $http_proxy    = undef,
```
Comment (bastelfreak) [inline manifests/init.pp:104]: did you remove those on purpose? Aren't they used anymore? Then we've to label it backwards-incompatible.

Comment (zilchms) [reply manifests/init.pp:104]: not on purpose and are definitly needed. i have no idea how i managed to bumble that in there

File: manifests/init.pp [hunk 2]
```diff
@@ -117,9 +117,7 @@
   Optional[Hash] $cas                      = undef,
 ) {
   # Is the package cron or systemd.timer based?
-  if ($facts['os']['name'] == 'Ubuntu' and versioncmp($facts['os']['release']['major'],'18.04') <= 0 ) or
-  ($facts['os']['name'] == 'Debian' and versioncmp($facts['os']['release']['major'],'10') <= 0 ) or
-  ($facts['os']['family'] == 'RedHat' and versioncmp($facts['os']['release']['major'],'7') <= 0 ) {
+  if $facts['os']['family'] == 'RedHat' and versioncmp($facts['os']['release']['major'],'7') <= 0 {
```
Comment (bastelfreak) [inline manifests/init.pp:120]: we could also drop EL7 since that's really dead soonish

Comment (zilchms) [reply manifests/init.pp:120]: true, but that we should do in a seperate PR :)

**Pull request merged**